stores/sql/sqlite/migrations/main/schema.sql

```diff
@@ -21,6 +21,7 @@ CREATE INDEX `idx_contracts_start_height` ON `contracts`(`start_height`);
 CREATE INDEX `idx_contracts_state` ON `contracts`(`state`);
 CREATE INDEX `idx_contracts_window_end` ON `contracts`(`window_end`);
 CREATE INDEX `idx_contracts_window_start` ON `contracts`(`window_start`);
+CREATE INDEX `idx_contracts_v2` ON `contracts`(`v2`);
```

Comment:
I don't think we ever query by the version so we can probably avoid the index.

Reply:
Feels (to me) like something we should index regardless but I'll remove it.